app/scripts/lib/transaction/metrics.ts

```diff
@@ -1133,6 +1141,30 @@ function getTransactionCompletionTime(submittedTime: number) {
   return Math.round((Date.now() - submittedTime) / 1000).toString();
 }
 
+/**
+ * Returns number of seconds (rounded to the hundredths) between submitted time
+ * and the block timestamp.
+ *
+ * @param submittedTime - The UNIX timestamp in milliseconds in which the
+ * transaction has been submitted
+ * @param blockTimestamp - The UNIX timestamp in seconds in hexadecimal in which
+ * the transaction has been confirmed in a block
+ */
+function getTransactionOnchainCompletionTime(
+  submittedTime: number,
+  blockTimestamp: string,
+): string {
+  const DECIMAL_DIGITS = 2;
```

Comment:
Minor, constants at top of file with extra context?

Reply:
Since this is only really going to be used in this function, I find it better to keep it as local constant, which makes it easier to understand its context too.